src/containers/BuySellCrypto/PaymentMethod/SendTransaction/SendTransaction.js

```diff
@@ -0,0 +1,145 @@
+            toCurr,
+            toVal,
+            buy
+        } = this.props.navigation.state.params;
+        this.state = {
+            paymentMethod,
+            fromCurr,
+            fromVal,
+            toCurr,
+            toVal,
+            buy
+        };
+      }
+
+    componentDidMount() {
+        this.props.getTransactions();
+    }
+
+
+    handleConfirm = () => {
+        this.props.sendTransaction(
+            this.state.paymentMethod,
+            this.state.fromCurr,
+            this.state.fromVal,
+            this.state.toCurr,
+            this.props.navigation,
+        );
+    }
+
+    back = () => {
+        this.props.navigation.dispatch(NavigationActions.back());
+    }
+
+    render() {
+        return(
+            <View style={styles.root}>
+                <View style={styles.containerConfirmTransaction}>
+                    <View style={styles.containerTransactionInfo}>
+                        <Text style={styles.textStyle}>
+                            {`Payment method: ${this.state.paymentMethod.name}`}
+                        </Text>
+                        <Text style={styles.textStyle}>
+                            {this.state.buy ? 
+                                `Buy ${this.state.toVal} "${this.state.toCurr}"`
+                                :
+                                `Sell ${this.state.fromVal} "${this.state.fromCurr}"`
+                            }
+                        </Text>
+                        <Text style={styles.textStyle}>
+                            {this.state.buy ? 
+                                `For ${this.state.fromVal} "${this.state.fromCurr}"`
+                                :
+                                `For ${this.state.toVal} "${this.state.toCurr}"`
+                            }
+                        </Text>
+                    </View>    
+                    <View style={styles.buttonContainer}>
+                        <Button1
+                            style={styles.saveChangesButton}
+                            buttonContent="Confirm"
+                            onPress={this.handleConfirm}
+                        />
+                        <Button1
+                            style={styles.backButton}
+                            buttonContent="Cancel"
+                            onPress={this.back}
+                        />
+                    </View>
+                </View>
+                <Text style={styles.textTransactionHistory}>Transaction history:</Text>
+                <ScrollView>
+                    <View style={styles.containerTransactionHistory} >
+                        {
+                            this.props.history && this.props.history.map((item) => (
+                                <View key={item.id}>
+                                    <ListItem
+                                        title={`${item.sourceCurrency==='USD' ? 'Bought' : 'Sold' } ${item.sourceAmount} ${item.sourceCurrency}`}
```

Comment:
This condition is invalid. Remove this bought sold based on the sourceCurrency condition.